enforcer_cached.go

```diff
@@ -80,7 +80,7 @@ func (e *CachedEnforcer) getCachedResult(key string) (res bool, ok bool) {
 	e.locker.RLock()
 	defer e.locker.RUnlock()
 	res, ok = e.m[key]
-	return res, ok
+	return
```

Comment:
Revert this.

Reply:
No, its okay, please check the function signature it already returns the parameters, the following return params are redundant.

Reply:
```
func (e *CachedEnforcer) getCachedResult(key string) (res bool, ok bool)
```
Please check this.

Reply:
I never commit and push syntactical errors in my code.